.gitignore

```
@@ -1,4 +1,4 @@
 .env
 __pycache__
-.venv
+.hwvenv
 .idea
```

gitignore лучше сюда же добавить
```suggestion
.idea
.gitignore
```
по большому счёту, он в удаленном репозитории не нужен